frontend/src/components/icons/ClapWhite.tsx

```diff
@@ -0,0 +1,71 @@
+        </g>{' '}
+      </g>
+    </svg>
+  );
+};
+
+const ClapIconWhite = () => {
+  const iconStyle: any = {
+    fill: 'none',
+    stroke: '#000000',
+    strokeWidth: 0,
+    strokeLinecap: 'round',
+    strokeLinejoin: 'round',
+  };
+  return (
+    <svg
+      version="1.1"
+      id="Capa_1"
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="0 0 285.247 285.247"
+      width="20px"
+      height="20px"
+      fill="#FFFFFF"
+      transform="matrix(-1, 0, 0, 1, 0, 0)"
+    >
+      <g id="SVGRepo_bgCarrier" strokeWidth="0"></g>
+      <g id="SVGRepo_tracerCarrier" strokeLinecap="round" strokeLinejoin="round"></g>
+      <g id="SVGRepo_iconCarrier">
+        {' '}
+        <g>
+          {' '}

```

Comment:
Even though this works fine, we can achieve the goal in a much cleaner way.

Instead of creating a new ClapWhite icon, you can re-use the existing icon and add classes for dark mode inside it.

Reply:
Yes. It can improve. Will try making reusable components